template/{{app_name}}/app/controllers/users/accounts_controller.rb

```diff
@@ -28,6 +26,16 @@ def update_email
   end
 
   private
+
+    def set_email_form
+      email_params = action_name == "update_email" ? user_email_params : { email: current_user.email }
```

Comment:
Hmm, this seems a little smelly. Is it common practice in Rails to have action specific behavior conditional in helpers?

I'd maybe leave the different `@email_form` building as is in each action, and then have the common `set_password_form` helper.